src/main/java/it/gov/pagopa/payhub/activities/util/Utilities.java

```diff
@@ -88,14 +86,20 @@ public static XMLGregorianCalendar toXMLGregorianCalendar(OffsetDateTime offsetD
         return offsetDateTime != null ? DATATYPE_FACTORY_XML_GREGORIAN_CALENDAR.newXMLGregorianCalendar(GregorianCalendar.from(offsetDateTime.toZonedDateTime())) : null;
     }
 
-
     public static OffsetDateTime toOffsetDateTime(XMLGregorianCalendar xmlGregorianCalendar) {
-        if(xmlGregorianCalendar == null) {
-            return null;
-        }
+        return Optional.ofNullable(xmlGregorianCalendar)
```

Comment:
let's avoid to introduce the usage of Optional.ofNull just to avoid to write if(... == null) return null.
let's write the if instead